content/tutorials/token-authentication.textile

```diff
@@ -828,17 +841,17 @@ blang[swift].
 
   bc[sh]. pod install
 
-  To learn more about using CocoaPods in your project visit "official CocoaPods guide":https://guides.cocoapods.org/using/using-cocoapods.html.
+  To learn more about using CocoaPods in your project visit the "official CocoaPods guide":https://guides.cocoapods.org/using/using-cocoapods.html.
 
-  Then in your files in which you will be using Ably import:
+  Then within your files which will be using Ably import:
```

Comment:
This currently signifies the files are using Ably import vs. the meaning is more towards the reader using it. Hence, probably best to retain earlier version by just replacing 'in' with 'within'